src/Categories/index.js

```diff
@@ -1,59 +1,60 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
-import { List, ListItem, ListItemText, Typography } from "@material-ui/core";
-import startCase from "lodash/startCase";
+import { Tabs, Tab } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import purple from "@material-ui/core/colors/purple";
+import grey from "@material-ui/core/colors/grey";
+import cyan from "@material-ui/core/colors/cyan";
 
 const useStyles = makeStyles((theme) => ({
-  heading: {
-    paddingLeft: theme.spacing(2),
+  tabText: {
+    color: grey[900],
     marginTop: theme.spacing(2),
-    fontFamily: "ChronicleDisp-Roman",
-    fontWeight: "bold"
+    textTransform: "capitalize",
+    fontSize: "1.5rem",
   },
   selected: {
-    color: purple[500],
-    fontWeight: "bold"
-  }
+    borderBottom: `${cyan[300]} 4px solid`,
+  },
 }));
 
 export const Categories = ({ handleFilterChange, categoryList }) => {
   const classes = useStyles();
-  const [categoryTitle, setCategoryTitle] = useState(null);
+  const [categoryIndex, setCategoryIndex] = useState(0);
```

Comment:
Sorry I know you were following my bad example but could we call  this `selectedCategoryIndex`? Just makes it a bit clearer

Reply:
Actually this wreaks havoc with the API calls - ignore